### PR TITLE
Fix ons select

### PR DESCRIPTION
**Pull request by misterjunio**
@frankdiox @asial-matagawa I provided a `select-id` attribute so that users can target the inner select, should be good to merge

### CODE REVIEW

File: core/src/elements/ons-select.js [hunk 1]
```diff
@@ -122,6 +123,14 @@ export default class SelectElement extends BaseElement {
    */
 
   /**
+   * @attribute select-id
+   * @type {String}
+   * @description
+   *   [en]ID given to the inner select, useful for dynamic manipulation.[/en]
+   *   [ja][/ja]
```
Comment (asial-matagawa) [inline core/src/elements/ons-select.js]: @misterjunio 
Nice work!
Could you add this Japanese doc here?
```
[ja]このセレクトボックスが内部に持つ select 要素に与える ID を指定します。セレクトボックスの内容を動的に変更する必要がある場合に使用します。[/ja]
```

Comment (misterjunio) [inline core/src/elements/ons-select.js]: Done

**Comment by asial-matagawa** [comment]
@misterjunio 
I forgot to tell you this:
- When you made changes (except for undocumented API and something) to the core or the bindings, please update `CHANGELOG.md` or `bindings/foo/CHANGELOG.md`.

Thank you.

File: core/src/elements/ons-select.js (hunk 1 above)
Comment (asial-matagawa) [inline core/src/elements/ons-select.js]: Thank you!

**Comment by frankdiox** [comment]
@misterjunio Looks great, thanks!

**Pull request merged**